arraystask/script.js

```diff
@@ -0,0 +1,89 @@
+function isArray1(a) {
+  return Array.isArray(a);
```

Comment:
this function doesn't make sense, the task is to implement your own `Array.isArray` and you did with the next 2 functions.